### PR TITLE
Skip email verification if disabled for the institution

### PR DESCRIPTION
**Pull request by jorissteyn**
The default behaviour of stepup is to require email verification. When
the user has proven possession of a second factor, selfservice
triggers the 'possession proven' command, which is handled by
middleware as follows:
    
 - an unverified second factor is created
 - a verification email is sent
 - user clicks on the verification link in the email
 - the unverified second factor transitions to verified second factor
 - an email containing the registration code is sent
    
This transition is fundamental to stepup and not easily changed. To
allow for users to skip email verification without having conditional
logic in many places, a second factor still goes trough the same
transition when email verification is disabled. When possession of a
second factor is proven the following happens in the same request:
    
 - an unverified second factor is created
 - no verification email is sent
 - the unverified second factor transitions to verified second factor
 - an email containing the registration code is sent

### CODE REVIEW

File: src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Validator/Fixtures/invalid_reconfigure_institution_request/missing_allowed_second_factors_option.php [hunk 1]
```diff
@@ -23,6 +23,7 @@
         'surfnet.nl' => [
             'use_ra_locations' => true,
             'show_raa_contact_information' => true,
+            'verify_email' => false,
```
Comment (MKodde) [inline src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Validator/Fixtures/invalid_reconfigure_institution_request/missing_allowed_second_factors_option.php]: Why did you set verify_email to false by default in the `invalid_reconfigure_institution_request` fixtures?

Comment (jorissteyn) [reply src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Validator/Fixtures/invalid_reconfigure_institution_request/missing_allowed_second_factors_option.php]: It's just an arbitrary value.

File: docs/postman/2.json [hunk 2]
```diff
@@ -1,232 +1,447 @@
...
+						"value": "application/json"
+					},
+					{
+						"key": "Authorization",
+						"value": "Basic bWFuYWdlbWVudDpiYXI="
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"dev.organisation.example\": {\n        \"use_ra_locations\": true, \n        \"show_raa_contact_information\": true,\n        \"verify_email\": false,\n        \"allowed_second_factors\": []\n    }\n}"
```
[the remark this answers is not in the document]
Comment (jorissteyn) [reply docs/postman/2.json]: Changed, the request now enables it by default.

[the remark this answers is not in the document]
Comment (jorissteyn) [reply docs/postman/2.json]: Changed, the request now enables it by default.

**Pull request merged**